lib/vanagon/project.rb

```diff
@@ -665,6 +669,28 @@ def save_manifest_json
       end
     end
 
+    # Writes a yaml file at `output/<name>-<version>.<platform>.settings.yaml`
+    # containing settings used to build the current project on the platform
+    # provided (and a corresponding sha1sum file) if `yaml_settings` has been
+    # set in the projcet definition.
+    #
+    # @param [Vanagon::Platform] the platform to publish settings for
+    def publish_yaml_settings(platform)
+      return unless yaml_settings
+      raise(Vanagon::Error, "You must specify a project version") unless version
+
+      filename = "#{name}-#{version}.#{platform.name}.settings.yaml"
+      filepath = File.join('output', filename)
+
+      File.open(filepath, 'w') do |f|
+        f.write(@settings.to_yaml)
+      end
+
+      File.open("#{filepath}.sha1", 'w') do |f|
```

Comment:
Any particular reason for sha1? I think we should be defaulting to sha256 these days. ( I realize the collision changes here are low, but just for uniformity)

Reply:
I was working with the assumption that `platform.shasum` used sha1sum by default -- do you (or does anyone) know if there's a way to confidently find a sha256sum executable for any platform?

Reply:
I can't recall if there was a reason I went with sha1 instead of sha256 when I was adding this functionality for runtime stuff, but it may have been related to platform compatibility? That seems like something to look into, but probably out of the scope of this work